src/main/java/Functional.java

```diff
@@ -0,0 +1,28 @@
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class Functional {
+    static int sum(List<Integer> list) {
+        int sum = 0;
+        for (int i : list) {
+            sum += i;
+        }
+        return sum;
+    }
+
+    static int max(List<Integer> list)  throws NoSuchElementException {
```

Comment:
It is good practice to specify all possible exceptions explicitly=)
However, remember that this particular exception is a subclass of `RuntimeException`, thus we may not specify it.